src/routing/FormRouter.tsx

```diff
@@ -17,12 +17,14 @@ import { PageContextProvider } from '../form-data/PageContext';
  */
 export default function FormRouter(props: FormRouterProps): JSX.Element {
   const initialValues = props.formData;
+  const uiInitialValues = props?.uiFormData || {};
+  const formValues = { ...initialValues, ...uiInitialValues };
```

Comment:
Should we add some tests around this logic to make sure we are getting the correct values?